components/Text.js

```diff
@@ -0,0 +1,85 @@
+import React from 'react'
```

Comment:
I'm going to touch this up a bit. Thanks for creating this though!

Reply:
Looks much better now, thanks for the touch-ups!